src/Simulation/Native/W15a_15.log

```diff
@@ -0,0 +1,95 @@
+@@@DBG: Generic
```

Comment:
There are LOTS of temp files checked in. Let me know what branch to go to (when you're ready) and I'll prune them.